.github/workflows/nightly.yaml

```diff
@@ -168,3 +168,48 @@ jobs:
         with:
           status: ${{ job.status }}
           fields: repo,workflow
+
+  go-get-test:
+    name: Go Get Smoke Test
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - id: go_version
+        name: Read go version
+        run: echo "go_version=$(cat .go-version)" >> $GITHUB_OUTPUT
+
+      - name: Install Go (${{ steps.go_version.outputs.go_version }})
+        uses: actions/setup-go@44694675825211faa026b3c33043df3e48a5fa00 # v6.0.0
+        with:
+          go-version: ${{ steps.go_version.outputs.go_version }}
+
+      - name: Test go get without proxy
+        run: |
```

Comment:
What's the exit status of the first `go get` in the pipeline if something goes wrong? I wonder if we should `set -o pipefail` here...